src/views/HomeView/index.js

```diff
@@ -3,7 +3,7 @@ import React, { PropTypes } from 'react'
 import { connect } from 'react-redux'
 import { increment, doubleAsync } from '../../redux/modules/counter'
 import DuckImage from './Duck.jpg'
-import classes from './HomeView.scss'
+import classes from './style.scss'
```

Comment:
Yep, I know that BEM was created to solve global CSS namespace, but for me it's just cleaner to use it even with CSS modules. Because of CSS modules I usually drop .Element--modifier convention and just add class .modifier